interface/HttSettings.h

```diff
@@ -392,6 +392,11 @@ class HttSettings: public KappaSettings {
 	IMPL_SETTING_STRINGLIST_DEFAULT(TauTrigger2017IDTypes, {});
 	IMPL_SETTING_STRINGLIST_DEFAULT(TauTrigger2017EfficiencyWeightNames, {});
 
+	// settings for TauIDEfficiencyProducer
```

Comment:
The name is TauIDScaleFactorProduce, right? Please change the comment accordingly

Reply:
Done.